---
layout: page
title: Publications
permalink: /publications/
---

\* equal contribution

* Gyaltshen Y.\*, **Rozenberg A.**\*, Paasch A., Burns J., Warring S., Larson R., et al. (2023) Long-read-based genome assembly reveals numerous endogenous viral elements in the green algal bacterivore *Cymbomonas tetramitiformis*. Genome Biology and Evolution, in revision. (**bioinformatic analyses of viral elements, genome annotation**).

* Mannen K., Nagata T., **Rozenberg A.**, Konno M., del Carmen Marín M., Bagherzadeh R., et al. (2023) Multiple roles of a conserved glutamate residue for unique biophysical properties in a new group of microbial rhodopsins homologous to TAT rhodopsin. Journal of Molecular Biology, accepted. (**data mining, metagenomic analysis, phylogenetic analysis**).

* Oppermann J., **Rozenberg A.**, Fabrin T., Gonzalez Cabrera C., Béjà O., Prigge M., Hegemann P. (2023) Robust optogenetic inhibition with red-light-sensitive anion-conducting channelrhodopsins. Elife, [preprint](https://elifesciences.org/reviewed-preprints/90100). (**data mining, phylogenetic analysis**).

* Bar-Shalom R.\*, **Rozenberg A.**\*, Lahyani M., Hassanzadeh B., Sahoo G., Haber M., Burgsdorf I., Tang X., Squatrito V., Gomez-Consarnau L., Béjà O., Steindler L. (2023) Rhodopsin-mediated nutrient uptake by cultivated photoheterotrophic *Verrucomicrobiota*. *ISME Journal* 17: 1063--1073 [doi:10.1038/s41396-023-01412-1](https://doi.org/10.1038/s41396-023-01412-1) (**shared first, genomics, phylogenetic analyses, metagenomics**)

* Hanna R., **Rozenberg A.**, Saied L., Ben-Yosef D., Lavy T., Kleifeld O. (2023) In-depth characterization of apoptosis N-terminome reveals a link between caspase-3 cleavage and posttranslational N-terminal acetylation. *Molecular & Cellular Proteomics* 22: 100584 [doi:10.1016/j.mcpro.2023.100584](https://doi.org/10.1016/j.mcpro.2023.100584) (**programming**)

* Chazan A.\*, Das I.\*, Fujiwara T.\*, Murakoshi S.\*, **Rozenberg A.**\*, Molina-Márquez A., Sano F., Tanaka T., Gómez-Villegas P., Larom S., Pushkarev A., Malakar P., Hasegawa M., Tsukamoto Y., Ishizuka T., Konno M., Nagata T., Mizuno Y., Katayama K., Abe-Yoshizumi R., Ruhman S., Inoue K., Kandori H., León R., Shihoya W., Yoshizawa S., Sheves M., Nureki O., Béjà O. (2023). Phototrophy by antenna-containing rhodopsin pumps in aquatic environments. *Nature* 615: 535--540 [doi:10.1038/s41586-023-05774-6](https://doi.org/10.1038/s41586-023-05774-6) (**shared first, bioinformatic analyses**)

* Roitman S., **Rozenberg A.**, Lavy T., Brussaard C.P.D., Kleifeld O., Béjà O. (2023) Isolation and infection cycle of a polinton-like virus virophage in an abundant marine alga. *Nature Microbiology* 8: 332--346 [doi:10.1038/s41564-022-01305-7](https://doi.org/10.1038/s41564-022-01305-7) (**sequencing, genome assembly, remote homology, phylogeny**)
	
* Vierock J., Peter E., Grimm C., **Rozenberg A.**, Chen I., Tillert L., Castro Scalise A., Casini M., Augustin S., Tanese D., Forget B., Peyronnet R., Schneider-Warme F., Emiliani V., Béjà O., Hegemann P. (2022) WiChR, a highly potassium-selective channelrhodopsin for low-light one- and two-photon inhibition of excitable cells. *Science Advances* 8(49): eadd7729 [doi:10.1126/sciadv.add7729](https://doi.org/10.1126/sciadv.add7729) (**data collection, phylogeny**)

* **Rozenberg A.**\*, Kaczmarczyk I.\*, Matzov D.\*, Vierock J.\*, Nagata T., Sugiura M., Katayama K., Kawasaki Y., Konno M., Nagasaka Y., Aoyama M., Das I., Pahima E., Church J., Adam S., Borin V.A., Chazan A., Augustin S., Wietek J., Dine J., Peleg Y., Kawanabe A., Fujiwara Y., Yizhar O., Sheves M., Schapiro I., Furutani Y., Kandori H., Inoue K., Hegemann P., Béjà O., Shalev-Benami M. (2022) Rhodopsin-bestrophin fusion proteins from unicellular algae form gigantic pentameric ion channels. *Nature Structural & Molecular Biology* 29:592--603 [doi:10.1038/s41594-022-00783-x](https://doi.org/10.1038/s41594-022-00783-x) (**initial discovery, bioinformatic analyses, project coordination, writing**)

* Hososhima S., Mizutori R., Abe-Yoshizumi R., **Rozenberg A.**, Shigemura S., Pushkarev A., Konno M., Katayama K., Inoue K., Tsunoda S.P., Béjà O., Kandori H. (2022) Proton-transporting heliorhodopsins from marine giant viruses. *eLife* 11:e78416 [doi:10.7554/eLife.78416](https://doi.org/10.7554/eLife.78416) (**phylogeny, bioinformatic support**)
	
* Chazan A.\*, **Rozenberg A.**\*, Mannen K., Nagata T., Tahan R., Yaish S., Larom S., Inoue K., Béjà O., Pushkarev A. (2022) Diverse heliorhodopsins detected via functional metagenomics in freshwater *Actinobacteria*, *Chloroflexi* and *Archaea*. *Environmental Microbiology* 24:110--121 [doi:10.1111/1462-2920.15890](https://doi.org/10.1111/1462-2920.15890) (**shared first, assembly, annotation, statistical analyses, writing**)

* **Rozenberg A.**\*, Inoue K.\*, Kandori H., Béjà O. (2021) Microbial rhodopsins: the last two decades. *Annual Review of Microbiology* 75:427--447 [doi:10.1146/annurev-micro-031721-020452](https://doi.org/10.1146/annurev-micro-031721-020452)

* Sahu I., Mali S., Sulkshane P., Xu C., **Rozenberg A.**, Morag R., Sahoo M., Singh S., Ding Z., Wang Y., Day S., Cong Y., Kleifeld O., Brik A., Glickman M. (2021) The 20S as a stand-alone proteasome in cells can degrade the ubiquitin tag. *Nature Communications* 12: 6173 [doi:10.1038/s41467-021-26427-0](https://doi.org/10.1038/s41467-021-26427-0) (**statistical analyses**)

* Bock N.A., Charvet S., Burns J., Gyaltshen Y., **Rozenberg A.**, Duhamel S., Kim E. (2021) Experimental identification and in silico prediction of bacterivory in green algae. *The ISME Journal* 15(7):1987--2000 [doi:10.1038/s41396-021-00899-w](https://doi.org/10.1038/s41396-021-00899-w) (**data collection**)

* **Rozenberg A.**, Oppermann J., Wietek J., Fernandez Lahore R.G., Sandaa R.-A., Bratbak G., Hegemann P., Béjà O. (2020) Lateral gene transfer of anion-conducting channelrhodopsins between green algae and giant viruses. *Current Biology* 30(24):4910--4920 [doi:10.1016/j.cub.2020.09.056](https://doi.org/10.1016/j.cub.2020.09.056) (**initial discovery, genome assembly, bioinformatic analyses**)
	
* Dömel J.S., Dietz L., Macher T.-H., **Rozenberg A.**, Mayer C., Spaak J.M., Melzer R.R., Leese F. (2020) Analyzing drivers of speciation in the Southern Ocean using the sea spider species complex *Colossendeis megalonyx* as a test case. *Polar Biology* 43(4):319--342 [doi:10.1007/s00300-020-02636-z](https://doi.org/10.1007/s00300-020-02636-z) (**statistical analyses**)
	
* Dömel J.S., Macher T.-H., Dietz L., Duncan S., Mayer C., **Rozenberg A.**, Wolcott K., Leese F., Melzer R.R. (2019) Combining morphological and genomic evidence to resolve species diversity and study speciation processes of the *Pallenopsis patagonica* (Pycnogonida) species complex. *Frontiers in Zoology* 16:36 [doi:10.1186/s12983-019-0316-y](https://doi.org/10.1186/s12983-019-0316-y) (**statistical analyses**)
	
* Nadel O., **Rozenberg A.**, Flores-Uribe J., Larom S., Schwarz R., Béjà O. (2018) An uncultured marine cyanophage encodes an active phycobilisome proteolysis adaptor protein NblA. *Environmental Microbiology Reports* 11:848--854 [doi:10.1111/1758-2229.12798](https://doi.org/10.1111/1758-2229.12798) (**bioinformatic analyses**)

* Macher J.N., Leese F., Weigand A.M., **Rozenberg A.** (2017) The complete mitochondrial genome of a cryptic amphipod species from the *Gammarus fossarum* complex. *Mitochondrial DNA Part B* 2:17--18 [doi:10.1080/23802359.2016.1275844](https://doi.org/10.1080/23802359.2016.1275844) (**assembly, phylogenetic analysis**)

* Dzyubenko E., **Rozenberg A.**, Hermann D.M., Faissner A. (2016) Colocalization of synapse marker proteins evaluated by STED-microscopy reveals patterns of neuronal synapse distribution *in vitro*. *Journal of Neuroscience Methods* 273:149--159 [doi:10.1016/j.jneumeth.2016.09.001](https://doi.org/10.1016/j.jneumeth.2016.09.001) (**programming**)

* **Rozenberg A.**, Leese F., Weiss L.C., Tollrian R. (2016) Digital gene expression analysis with sample multiplexing and PCR duplicate detection: A straightforward protocol. *BioTechniques* 61:26--32 [doi:10.2144/000114434](https://doi.org/10.2144/000114434) (**wet-lab work, programming, data analysis, writing**)

* **Rozenberg A.**, Brand P., Rivera N., Leese F., Schubart C.D. (2015) Fossilized relatives of the White Spot Syndrome Virus (WSSV) in decapod genomes. *BMC Evolutionary Biology* 15:142 [doi:10.1186/s12862-015-0380-7](https://doi.org/10.1186/s12862-015-0380-7) (**bioinformatic analyses, wet-lab work, writing**)

* **Rozenberg A.**, Parida M., Leese F., Weiss L., Tollrian R., Manak J.R. (2015). Transcriptional profiling of predator-induced phenotypic plasticity in *Daphnia pulex*. *Frontiers in Zoology* 12:18 [doi:10.1186/s12983-015-0109-x](https://doi.org/10.1186/s12983-015-0109-x) (**wet-lab work, differential expression analysis, writing**)	

* Macher J.M., **Rozenberg A.**, Pauls S.U., Tollrian R., Wagner R., Leese F. (2015) Assessing the phylogeographic history of the montane caddisfly *Thremma gallicum*: a comparison of sequence data from mitochondria and restriction-site associated DNA markers (RAD). *Ecology and Evolution* 5:648--662 [doi:/10.1002/ece3.1366](https://doi.org/10.1002/ece3.1366) (**statistical analyses**)

* Schweyen H., **Rozenberg A.**, Leese F. (2014) Detection and removal of PCR duplicates in population genomic ddRAD studies by addition of a degenerate base region (DBR) in sequencing adapters. *Biological Bulletin* 227:146--160 [doi:10.1086/BBLv227n2p146](https://doi.org/10.1086/BBLv227n2p146) (**project design, analyses, writing**)

* Leese F., Brand P., **Rozenberg A.** et al (2012) Exploring Pandora's Box: Potential and pitfalls of low coverage genome surveys for evolutionary biology. *PLoS ONE* 7:e49202 [doi:10.1371/journal.pone.0049202](https://doi.org/10.1371/journal.pone.0049202) (**bioinformatic analyses**)

* Tchesunov A.V., **Rozenberg A.A.** (2011) Data on the life cycle of parasitic benthimermithid nematodes with the description of a new species discovered in marine aquaria. *Russian Journal of Nematology* 19:139--150 (**initial discovery, field work**)

* Tchesunov, A.V., **Rozenberg A.A.**, Aleshin, V.V. (2009). Position of Benthimermithidae (parasites of marine invertebrates) as derived from morphological and molecular data. *Russian Journal of Nematology* 17: 162.

* Khaitov V.M., **Rozenberg A.A.**, Terovskaya E.V. (2007) Morphological differences between females of different *Jaera* species (Crustacea: Asellota: Isopoda) in the White Sea: a possible solution to an old problem. *Marine Biology* 150:1205--1214 [doi:10.1007/s00227-006-0450-x](https://doi.org/10.1007/s00227-006-0450-x) (**experimental work, statistical analysis**)
